crates/polars-io/src/csv/read/splitfields.rs

```diff
@@ -360,12 +360,12 @@ mod inner {
                                 .unwrap_unchecked_release()
                         };
                         let simd_bytes = SimdVec::from(lane);
-                        let has_eol_char = simd_bytes.simd_eq(self.simd_eol_char).to_bitmask();
-                        let has_separator = simd_bytes.simd_eq(self.simd_separator).to_bitmask();
-                        let has_any = has_separator | has_eol_char;
+                        let has_eol_char = simd_bytes.simd_eq(self.simd_eol_char);
+                        let has_separator = simd_bytes.simd_eq(self.simd_separator);
+                        let has_any_mask = (has_separator | has_eol_char).to_bitmask();
```

Comment:
You rather do that on the SIMD lane than the `u64`?

Reply:
Yes, `to_bitmask` is a bit more expensive on ARM, so I'd rather do that only once.